templates/dex-k8s-authenticator.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: dex-k8s-authenticator
+  namespace: kubeaddons
+spec:
+  kubernetes:
+    minSupportedVersion: v1.14.0
+  chartReference:
+    chart: raw.githubusercontent.com/dex-k8s-authenticator
+    # TODO(mh): If this is going to be used figure out the production URL
+    # See: https://github.com/mintel/dex-k8s-authenticator/issues/70
+    repo: https://raw.githubusercontent.com/mintel/dex-k8s-authenticator/feat-helm-repo/charts/repo/stable
+    version: 1.0.0
```

Comment:
Shouldn't the version be `version: 1.1.1` as in mesosphere/charts ?

Reply:
yes, it has to be that version https://github.com/mesosphere/charts/blob/master/docs/staging/index.yaml#L21

Reply:
Yes, I am still working on an update to this PR where I'll point dex-k8s-authenticator to our custom chart repo and as well move use dex container from `mesosphere/dex` repo.

Reply:
👍